lib/pdk/validators/util/task_metadata_schema.rb

```diff
@@ -0,0 +1,65 @@
+module PDK
```

Comment:
Is it possible to have the schema as a separate file instead of embedded?  That way other tools can consume it

Reply:
Also means the schema file itself can be validated

Reply:
There's been some discussion on tech-discuss and I've proposed hosting a JSON Schema for task (and module) metadata on the Forge. I've got a PR for that with just a placeholder so we could move this schema into that PR:

https://github.com/puppetlabs/puppet-forge-api/pull/371

Reply:
Alternatively we could publish at http://schemastore.org/json/

Reply:
![](https://vignette.wikia.nocookie.net/rwby/images/4/45/82733-why-not-both-meme-5LvD.jpeg/revision/latest?cb=20160618185223)

Reply:
Because two sources of truth is rarely a good thing.  Pick one.

Reply:
Then I'll easily pick the one we have direct control over.

Seems like the best approach would be to host it ourselves and just add it to the schemastore catalog.

Reply:
Done.